index.html

```diff
@@ -6,14 +6,50 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Ojemba | Team 5</title>
   </head>
-  <body>
+  <body class="overflow-x-hidden">
     <header id="header"></header>
     <main id="main"> My Main
       <section>
+        <h1 class="after:h-5 after:w-10 after:bg-primary">let's see if this will work</h1>
         <div id="expand"></div>
       </section>
+      <section id="skillset-section" class="bg-gradient-to-r from-secondary to-primary space-y-14 md:space-y-28">
+        <div id="skillset" class=" text-white p-4 space-y-8">
+          <div id="textWithLine" class="relative h-[46px] w-[564px]">
```

Comment:
please revise how you designed the line ,cause it appears like it is oblique  instead of being horizontal